pkg/cmd/codespace/jupyter.go

```diff
@@ -87,3 +87,26 @@ func (a *App) Jupyter(ctx context.Context, codespaceName string) (err error) {
 		return nil // success
 	}
 }
+
+func connectToGRPCServer(ctx context.Context, session liveshareSession, token string) (*grpc.Client, error) {
+	ctx, _ = context.WithTimeout(ctx, grpc.ConnectionTimeout)
```

Comment:
Discarding the cancel here still seems to make the linter unhappy:
```
pkg/cmd/codespace/jupyter.go:92:7: lostcancel: the cancel function returned by context.WithTimeout should be called, not discarded, to avoid a context leak (govet)
        ctx, _ = context.WithTimeout(ctx, grpc.ConnectionTimeout)
             ^
```

Reply:
I think i have a fix for this, ill push it as soon as I can

Reply:
done, there's no need to ignore the cancel anymore since we are more concretely handling how to timeout

Reply:
Makes sense, thanks!